scripts/building/check_enforcer.py

```diff
@@ -0,0 +1,288 @@
+# Import necessary libraries
+import os
+import fnmatch
+import subprocess
+import time
+import argparse
+import json
+import sys
+
+# Define variables
+github_repository = "microsoft/promptflow"
+snippet_debug = 1  # Write debug info to console.
+merge_commit = ""
+loop_times = 30
+github_workspace = os.path.expanduser("~/promptflow/")
+
+# Special cases for pipelines that need to be triggered more or less than default value 1.
+# If 0, the pipeline will not be ignored in check enforcer.
+# Please notice that the key should be the Job Name in the pipeline.
+special_care = {
+    "sdk_cli_tests": 4,
```

Comment:
what's the integer here means? 4 and 0?

Reply:
sdk cli tests has 4 matrix triggers. python 3.8, 3.9. 3.10, 3.11.

This special_care number is you cannot trigger less than this number. 

I want these 4 runs all triggered. Then this is 4.
You can also set 3, then if over 3 sdk cli test is triggered, this check is passed.

if you set this value to 0, then no matter how much you are triggered, this check enforcer will just don't check this pipeline success or not.